test/exorcist.js

```diff
@@ -39,7 +40,7 @@ test('\nwhen piping a bundle generated with browserify through exorcist without
       t.equal(map.sources[0].indexOf(base), 0, 'uses absolute source paths')
       t.equal(map.sourcesContent.length, 4, 'includes 4 source contents')
       t.equal(map.mappings.length, 106, 'maintains mappings')
-      t.equal(map.sourceRoot, '', 'leaves source root an empty string')
+      t.equal(map.sourceRoot, '', 'if source root missing, emplaces empty string')
 
```

Comment:
"emplace", goddamn its nice to see someone who knows a dictionary. Top marks just for word usage! 👍